<!DOCTYPE html>

<!--[if lt IE 7]> <html class="no-js lt-ie9 lt-ie8 lt-ie7" lang="en"> <![endif]-->
<!--[if IE 7]>    <html class="no-js lt-ie9 lt-ie8" lang="en"> <![endif]-->
<!--[if IE 8]>    <html class="no-js lt-ie9" lang="en"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en"> <!--<![endif]-->
<head><meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
<meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
<meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="/xmlui/themes/Mirage2/images/favicon.ico" rel="shortcut icon"/>
<link href="/xmlui/themes/Mirage2/images/apple-touch-icon.png" rel="apple-touch-icon"/>
<meta content="DSpace 5.8" name="Generator"/>
<link href="https://fonts.googleapis.com/css?family=PT+Sans:400,700|Ubuntu:400,500,700italic|Droid+Sans:400,700" rel="stylesheet" type="text/css"/>
<link href="/xmlui/themes/Mirage2/styles/main.css" rel="stylesheet"/>
<link href="https://papyrus.bib.umontreal.ca:443/xmlui/description.xml" rel="search" title="DSpace" type="application/opensearchdescription+xml"/>
<script>
                //Clear default text of emty text areas on focus
                function tFocus(element)
                {
                if (element.value == ''){element.value='';}
                }
                //Clear default text of emty text areas on submit
                function tSubmit(form)
                {
                var defaultedElements = document.getElementsByTagName("textarea");
                for (var i=0; i != defaultedElements.length; i++){
                if (defaultedElements[i].value == ''){
                defaultedElements[i].value='';}}
                }
                //Disable pressing 'enter' key to submit a form (otherwise pressing 'enter' causes a submission to start over)
                function disableEnterKey(e)
                {
                var key;

                if(window.event)
                key = window.event.keyCode;     //Internet Explorer
                else
                key = e.which;     //Firefox and Netscape

                if(key == 13)  //if "Enter" pressed, then disable!
                return false;
                else
                return true;
                }
            </script><!--[if lt IE 9]>
                <script src="/xmlui/themes/Mirage2/vendor/html5shiv/dist/html5shiv.js"> </script>
                <script src="/xmlui/themes/Mirage2/vendor/respond/respond.min.js"> </script>
                <![endif]--><script src="/xmlui/themes/Mirage2/vendor/modernizr/modernizr.js"> </script><script src="https://www.bib.umontreal.ca/modele/urgence-udem/1.0.0/urgence-udem.js"></script>
<title>Accélération de l'exploration de l'espace chimique du cytochrome P450 BM3 par des méthodes de criblage à haut débit et bio-informatiques</title>
<meta content="Dr8axyf94eWBJ8HQjNW4eKxTumAAPsNbvRwpJYMbgGs" name="google-site-verification"/>
<link href="http://purl.org/dc/terms/" rel="schema.DCTERMS">
<link href="http://purl.org/dc/elements/1.1/" rel="schema.DC">
<meta content="Pelletier, Joelle" name="DC.contributor">
<meta content="Rousseau, Olivier" name="DC.creator">
<meta content="2019-05-24T16:50:00Z" name="DCTERMS.dateAccepted" scheme="DCTERMS.W3CDTF">
<meta content="NO_RESTRICTION" name="DCTERMS.available" scheme="DCTERMS.W3CDTF" xml:lang="fr">
<meta content="2019-05-24T16:50:00Z" name="DCTERMS.available" scheme="DCTERMS.W3CDTF">
<meta content="2019-03-13" name="DCTERMS.issued" scheme="DCTERMS.W3CDTF">
<meta content="2018-09" name="DC.date" scheme="DCTERMS.W3CDTF"/>
<meta content="http://hdl.handle.net/1866/21949" name="DC.identifier" scheme="DCTERMS.URI"/>
<meta content="Analyse de composantes indépendantes du temps et de la structure" name="DC.subject" xml:lang="fr"/>
<meta content="analyse statistique" name="DC.subject" xml:lang="fr"/>
<meta content="automatisation" name="DC.subject" xml:lang="fr"/>
<meta content="biocatalyse" name="DC.subject" xml:lang="fr"/>
<meta content="cétone de framboise" name="DC.subject" xml:lang="fr"/>
<meta content="criblage à haut débit" name="DC.subject" xml:lang="fr"/>
<meta content="cytochrome P450 BM3" name="DC.subject" xml:lang="fr"/>
<meta content="dynamique moléculaire" name="DC.subject" xml:lang="fr"/>
<meta content="indigo" name="DC.subject" xml:lang="fr"/>
<meta content="ingénierie de protéines" name="DC.subject" xml:lang="fr"/>
<meta content="modélisation" name="DC.subject" xml:lang="fr"/>
<meta content="mutagénèse" name="DC.subject" xml:lang="fr"/>
<meta content="oxydation" name="DC.subject" xml:lang="fr"/>
<meta content="séquençage nouvelle génération" name="DC.subject" xml:lang="fr"/>
<meta content="Automation" name="DC.subject" xml:lang="fr"/>
<meta content="Biocatalysis" name="DC.subject" xml:lang="fr"/>
<meta content="High-throughput screening" name="DC.subject" xml:lang="fr"/>
<meta content="Modeling" name="DC.subject" xml:lang="fr"/>
<meta content="Molecular dynamics" name="DC.subject" xml:lang="fr"/>
<meta content="Mutagenesis" name="DC.subject" xml:lang="fr"/>
<meta content="Next-generation sequencing" name="DC.subject" xml:lang="fr"/>
<meta content="Oxidation" name="DC.subject" xml:lang="fr"/>
<meta content="Protein engineering" name="DC.subject" xml:lang="fr"/>
<meta content="Raspberry ketone" name="DC.subject" xml:lang="fr"/>
<meta content="Statistical analysis" name="DC.subject" xml:lang="fr"/>
<meta content="Time-structure independent component analysis" name="DC.subject" xml:lang="fr"/>
<meta content="Chemistry - Biochemistry / Chimie - Biochimie (UMI : 0487)" name="DC.subject" xml:lang="fr"/>
<meta content="Accélération de l'exploration de l'espace chimique du cytochrome P450 BM3 par des méthodes de criblage à haut débit et bio-informatiques" name="DC.title" xml:lang="fr"/>
<meta content="Thèse ou mémoire / Thesis or Dissertation" name="DC.type"/>
<meta content="L’application de la chimie organique est prépondérante dans plusieurs secteurs industriels tels que le pharmaceutique, le cosmétique, l’alimentaire ou les produits ménagers. La chimie organique évolue sans cesse pour rendre les synthèses plus efficaces et moins dispendieuses. Cependant, l’amélioration de voies de synthèse se fait souvent sans égard à leur effet sur l’environnement. Ceci a entrainé l’émergence de la chimie verte pour réduire leur impact environnemental. À cette fin, l’une des stratégies employées est la biocatalyse : l’utilisation d’enzymes due à leurs propriétés ‘vertes’ et leur grande efficacité réactionnelle. 
Les oxydases de type cytochrome P450 sont reconnues pour leur grande promiscuité, liée à leur rôle dans la détoxification de composés xénobiotiques et de molécules endogènes dans le corps. Cette superfamille d’enzymes accomplit une diversité de réactions d’oxydations en une seule étape telles que l’hydroxylation, l’époxidation, la désamination, la déshalogénation et la cyclopropanation. La P450 BM3 (P450 BM3) de Bacillus megaterium possède l’une des plus grandes activités catalytiques de sa famille, justifiant l’intérêt grandissant de ses applications industrielles. Ce mémoire se penche sur la nécessité d’améliorer les essais afin d’augmenter la capacité de criblage pour obtenir une plus grande proportion de variants  capable de catalyser une réaction désirée. 
Il a été proposé que la synthèse de l’indigo par les variants de P450 BM3 soit un indicateur de promiscuité envers de nouveaux substrats. Nous avons exploré l’espace mutationnel de P450 BM3 pour la synthèse de l’indigo en comparant différentes méthodes de criblage de librairies de variants.  Dans un premier temps, des librairies de variants furent créées par mutagénèse par saturation de site. Cela permet d’obtenir une représentation uniforme des variants à chaque position sélectionnée plus rapidement et à un coût modique. Ensuite, des méthodes de criblage furent optimisées pour quantifier les variants de P450 BM3 et les cribler de deux façons. Nous avons comparé un essai colorimétrique direct grâce à l’absorbance de l’indigo avec un essai général indirect combinant la fluorescence du cofacteur NADPH et la spectrométrie de masse (LC-MS, GC-MS et LC préparative). Cette deuxième approche permet de cribler les variants indépendamment du substrat. Afin d’augmenter le débit du criblage, nous avons optimisé les méthodes en utilisant des plaques 96 puits avec une station de pipetage automatisée. Nous avons découvert de multiples nouveaux variants de P450 BM3 synthétisant l’indigo dont une forte fraction de ces derniers s’est révélée aptes à synthétiser une molécule de valeur élevée en industrie : la cétone de framboise. 
La corrélation de l’activité de P450 BM3 avec l’indigo et la cétone de framboise justifie l’exploration plus profonde de l’espace mutationnel de P450 BM3. La compréhension de cette corrélation permettrait de guider d’avantage l’ingénierie de P450 BM3 vers de nouveaux substrats. À cette fin, nous tentons d’expliquer la relation entre la structure, le dynamisme et l’activité des variants de P450 BM3. Les méthodes de criblage conventionnelles ne sont pas assez efficaces pour générer de grandes séries de données (« Big Data »). Nous appliquons le séquençage de nouvelle génération afin d’identifier &gt;104 variants simultanément pour relier le génotype (ADN) au phénotype (synthèse d’indigo). L’analyse de ces données nécessite l’utilisation d’outils statistiques et informatiques tels que la modélisation, la dynamique moléculaire et l’analyse de composantes indépendantes du temps et de la structure (tICA). Nous observons une tendance commune des variants actifs de P450 BM3 représenté par un changement d’équilibre entre ses deux conformations majeures.  Ce changement conformationnel pourrait expliquer la propension de P450 BM3 à synthétiser l’indigo.
Ce mémoire présente ainsi des méthodes permettant d’accélérer la découverte de variants actifs de P450 BM3 pour diverses réactions d’intérêt en industrie. Les méthodologies ci-développées pourront ensuite guider l’ingénierie d’autres systèmes enzymatiques." name="DCTERMS.abstract" xml:lang="fr"/>
<meta content="Organic chemistry plays a major role in the pharmaceutical, cosmetic, food and household product industries. Industrial organic synthesis has constantly evolved to be more efficient and less cost-effective. However, those improvements were often made regardless of their impact on the environment, spawning the creation of the discipline of Green Chemistry. One among many strategies to make chemistry more innocuous to the environment is to apply biocatalysis: the use of enzymes for their ‘green’ properties and catalytic efficacy.
Cytochrome P450 oxidases are well known for their substrate promiscuity in detoxification of xenobiotics and endogenous molecules in the body. This superfamily of enzymes catalyzes a diversity of oxidation reactions in a single step, including hydroxylations, epoxidations, deaminations, dehalogenations and cyclopropanations. Cytochrome P450 BM3 (P450 BM3) from Bacillus megaterium is one of the most efficient of its family, justifying the growing interest for its industrial applications. This thesis considers the necessity to improve assays in order to increase the throughput of enzyme library screening and to improve the ratio of active variants.
Indigo synthesis has been proposed to be an efficient predictor of promiscuity towards novel substrates of P450 BM3. We explore the mutational space of P450 BM3 for indigo synthesis and compare screening methods. We made variant libraries using site-directed saturation mutagenesis to obtain a uniform distribution of mutations at each position and reduce cost and time. We optimized a method to quantify P450 BM3 variants and two different screening methods. We compared a direct colorimetric assay using indigo absorbance and an indirect assay using NADPH fluorescence coupled with mass spectrometry (LC-MS, GC-MS and LC prep). This second approach circumvents the need to develop an assay dependent of the substrate. To increase screening throughput, we optimized methods in a 96-well plate format and use an automated pipetting station. We confirm the discovery of new indigo-producing variants. We further demonstrate that indigo can serve as a strong predictor of raspberry ketone production, a high value industrial compound.

The correlation of the activity of P450 BM3 with indigo and raspberry ketone justifies the exploration of the fitness landscape P450 BM3. Understanding this correlation would accelerate engineering of P450 BM3 for reaction with novel substrates. We attempt to explain the relationship between the structure, dynamism and activity through deep-mutational scanning of P450 BM3. Conventional screening methods are not sufficient to produce large datasets (Big Data); we apply next-generation sequencing (NGS) to identify &gt;104 variants simultaneously and to link the genotype (DNA) and the phenotype (indigo production).  Analysis of NGS for enzyme engineering required the development of statistical and bioinformatics tools such as modeling, molecular dynamics and time-structure independent component analysis (tICA). Those methods allowed us to observe conformational perturbations that were shared between the active variants represented as a shift of equilibrium between two major conformations.  This shift may explain the propensity of diverse P450 BM3 variants to synthesize indigo.
This master’s thesis successfully demonstrates the accelerated discovery of engineered variants of P450 BM3 with activity for industrially relevant reactions. The methodologies we have developed contribute to knowledge on enzyme engineering and have the potential to be applied more broadly in other enzyme systems." name="DCTERMS.abstract" xml:lang="fr"/>
<meta content="fra" name="DCTERMS.language" xml:lang="fr"/>
<meta content="Analyse de composantes indépendantes du temps et de la structure; analyse statistique; automatisation; biocatalyse; cétone de framboise; criblage à haut débit; cytochrome P450 BM3; dynamique moléculaire; indigo; ingénierie de protéines; modélisation; mutagénèse; oxydation; séquençage nouvelle génération; Automation; Biocatalysis; High-throughput screening; Modeling; Molecular dynamics; Mutagenesis; Next-generation sequencing; Oxidation; Protein engineering; Raspberry ketone; Statistical analysis; Time-structure independent component analysis; Chemistry - Biochemistry / Chimie - Biochimie (UMI : 0487); Thèse ou mémoire / Thesis or Dissertation; Chimie" name="citation_keywords"/>
<meta content="Accélération de l'exploration de l'espace chimique du cytochrome P450 BM3 par des méthodes de criblage à haut débit et bio-informatiques" name="citation_title"/>
<meta content="fra" name="citation_language"/>
<meta content="Rousseau, Olivier" name="citation_author"/>
<meta content="https://papyrus.bib.umontreal.ca/xmlui/bitstream/1866/21949/2/Rousseau_Olivier_2018_memoire.pdf" name="citation_pdf_url"/>
<meta content="2019-03-13" name="citation_date"/>
<meta content="https://papyrus.bib.umontreal.ca/xmlui/handle/1866/21949" name="citation_abstract_html_url"/>
<script type="text/x-mathjax-config">
                    MathJax.Hub.Config({
                      tex2jax: {
                        inlineMath: [['\\(','\\)']],
                        ignoreClass: "detail-field-data|detailtable|exception"
                      },
                      TeX: {
                        Macros: {
                          AA: '{\\mathring A}'
                        }
                      }
                    });
                </script><script src="//cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"> </script>
</meta></meta></meta></meta></meta></meta></link></link></head><body>
<header>
<div class="navbar navbar-default navbar-static-top" role="navigation">
<div class="container">
<div class="hidden-xs hidden-sm hidden-md" id="um-bandeau" role="banner">
<nav class="um-nav um-nav-hidden" role="navigation">
<span class="hidden">Liens externes</span>
<ul>
<li>
<a href="https://www.umontreal.ca/#udemwww-search-personne">Directories</a>
</li>
<li>
<a href="http://www.umontreal.ca/repertoires/facultes.html">Faculties</a>
</li>
<li>
<a href="http://www.bib.umontreal.ca/">Libraries</a>
</li>
<li>
<a href="http://plancampus.umontreal.ca/">Campus maps</a>
</li>
<li>
<a href="http://www.umontreal.ca/index/az.html">Sites A to Z</a>
</li>
<li>
<a>My UdeM</a>
<ul>
<li>
<a href="https://monudem.umontreal.ca/">My UdeM Portal</a>
</li>
<li>
<a href="https://outlook.umontreal.ca">My email</a>
</li>
<li>
<a href="https://studium.umontreal.ca/">StudiUM</a>
</li>
</ul>
</li>
</ul>
</nav>
<form action="https://google.com/cse" class="um-recherche" id="um-recherche" role="search">
<input name="cx" type="hidden" value="011926736769028447783:qlpu3so2kqq"/><input name="ie" type="hidden" value="ISO-8859-1"/><label class="hidden" for="um-boite-recherche">Search in : </label><span class="um-boite-bouton"><input class="um-boite" id="um-boite-recherche" maxlength="255" name="q" placeholder="UdeM" title="Type your search text" type="text" value=""/><input alt="Search in : " class="um-bouton" src="/xmlui/themes/Mirage2/images/bib-umontreal/loupe.gif" type="image"/></span>
</form>
</div>
<div class="hidden-xs hidden-sm visible-md visible-lg" id="papyrus-deco">
<img alt="Dessin du pavillon Roger Gaudry/Sketch of Roger Gaudry Building" border="0" height="89px" src="/xmlui/themes/Mirage2/images/UdeM.gif" title="Dessin du pavillon Roger Gaudry/Sketch of Roger Gaudry Building" width="193px"/></div>
<div class="navbar-header">
<button class="navbar-toggle" data-toggle="offcanvas" type="button"><span class="sr-only">Toggle navigation</span><span class="icon-bar"></span><span class="icon-bar"></span><span class="icon-bar"></span></button><a class="navbar-brand hidden-xs hidden-sm visible-md visible-lg um-logo-md-lg" href="http://www.umontreal.ca/english/index.html"><img alt="University Home page" src="/xmlui/themes/Mirage2/images/logo-UdeM.svg" title="Back to University Home page"/></a><a class="navbar-brand hidden-xs visible-sm hidden-md hidden-lg um-logo-sm" href="http://www.umontreal.ca/english/index.html"><img alt="University Home page" src="/xmlui/themes/Mirage2/images/logo-UdeM.svg" title="Back to University Home page"/></a><a class="navbar-brand visible-xs hidden-sm hidden-md hidden-lg um-logo-xs" href="http://www.umontreal.ca/english/index.html"><img alt="University Home page" src="/xmlui/themes/Mirage2/images/logo-UdeM.svg" title="Back to University Home page"/></a>
<div class="navbar-brand separateur visible-xs hidden-sm hidden-md hidden-lg um-sep-site-xs"></div>
<div class="navbar-brand separateur hidden-xs visible-sm hidden-md hidden-lg um-sep-site-sm"></div>
<div class="navbar-brand separateur visible-md visible-lg hidden-xs hidden-sm um-sep-site-md-lg"></div>
<div class="navbar-brand visible-md visible-lg hidden-xs hidden-sm um-titre-site-md-lg">
<span class="papyrus-signature">Papyrus</span> :
                                Institutional Repository</div>
<div class="navbar-brand hidden-xs visible-sm hidden-md hidden-lg um-titre-site-sm">
<span class="papyrus-signature">Papyrus</span>
<br/>Institutional Repository</div>
<div class="navbar-brand visible-xs hidden-sm hidden-md hidden-lg um-titre-site-xs">
<span class="papyrus-signature">Papyrus</span>
</div>
<div class="navbar-header pull-right visible-xs hidden-sm hidden-md hidden-lg">
<ul class="nav nav-pills pull-left">
<li class="dropdown" id="ds-language-selection-xs">
<button class="dropdown-toggle navbar-toggle navbar-link" data-toggle="dropdown" href="#" id="language-dropdown-toggle-xs" role="button"><b aria-hidden="true" class="visible-xs glyphicon glyphicon-globe"></b></button>
<ul aria-labelledby="language-dropdown-toggle-xs" class="dropdown-menu pull-right" data-no-collapse="true" role="menu">
<li role="presentation">
<a href="https://papyrus.bib.umontreal.ca:443/xmlui/handle/1866/21949?locale-attribute=fr&amp;show=full">français</a>
</li>
<li class="disabled" role="presentation">
<a href="https://papyrus.bib.umontreal.ca:443/xmlui/handle/1866/21949?locale-attribute=en&amp;show=full">English</a>
</li>
</ul>
</li>
<li>
<form action="/xmlui/login" method="get" style="display: inline">
<button class="navbar-toggle navbar-link"><b aria-hidden="true" class="visible-xs glyphicon glyphicon-user"></b></button>
</form>
</li>
</ul>
</div>
</div>
<div class="navbar-header pull-right hidden-xs">
<div class="nav-language-and-login-lg hidden-xs hidden-sm hidden-md">
<ul class="nav navbar-nav pull-right">
<li class="dropdown" id="ds-language-selection">
<a class="dropdown-toggle" data-toggle="dropdown" href="#" id="language-dropdown-toggle" role="button"><span class="hidden-xs">English <b class="caret"></b></span></a>
<ul aria-labelledby="language-dropdown-toggle" class="dropdown-menu pull-right" data-no-collapse="true" role="menu">
<li role="presentation">
<a href="https://papyrus.bib.umontreal.ca:443/xmlui/handle/1866/21949?locale-attribute=fr&amp;show=full">français</a>
</li>
<li class="disabled" role="presentation">
<a href="https://papyrus.bib.umontreal.ca:443/xmlui/handle/1866/21949?locale-attribute=en&amp;show=full">English</a>
</li>
</ul>
</li>
</ul>
<ul class="nav navbar-nav pull-right">
<li>
<a href="/xmlui/login"><span class="hidden-xs">Login</span></a>
</li>
</ul>
</div>
<div class="hidden-lg">
<ul class="nav navbar-nav pull-right">
<li class="dropdown" id="ds-language-selection">
<a class="dropdown-toggle" data-toggle="dropdown" href="#" id="language-dropdown-toggle" role="button"><span class="hidden-xs">English <b class="caret"></b></span></a>
<ul aria-labelledby="language-dropdown-toggle" class="dropdown-menu pull-right" data-no-collapse="true" role="menu">
<li role="presentation">
<a href="https://papyrus.bib.umontreal.ca:443/xmlui/handle/1866/21949?locale-attribute=fr&amp;show=full">français</a>
</li>
<li class="disabled" role="presentation">
<a href="https://papyrus.bib.umontreal.ca:443/xmlui/handle/1866/21949?locale-attribute=en&amp;show=full">English</a>
</li>
</ul>
</li>
</ul>
<ul class="nav navbar-nav pull-right">
<li>
<a href="/xmlui/login"><span class="hidden-xs">Login</span></a>
</li>
</ul>
</div>
<button class="navbar-toggle visible-sm" data-toggle="offcanvas" type="button"><span class="sr-only">Toggle navigation</span><span class="icon-bar"></span><span class="icon-bar"></span><span class="icon-bar"></span></button>
</div>
</div>
</div>
</header>
<div class="trail-wrapper hidden-print">
<div class="container">
<div class="row">
<div class="col-xs-12">
<div class="breadcrumb dropdown visible-xs">
<a class="dropdown-toggle" data-toggle="dropdown" href="#" id="trail-dropdown-toggle" role="button">View Item <b class="caret"></b></a>
<ul aria-labelledby="trail-dropdown-toggle" class="dropdown-menu" role="menu">
<li role="presentation">
<a href="/xmlui/" role="menuitem"><i aria-hidden="true" class="glyphicon glyphicon-home"></i>  Home</a>
</li>
<li role="presentation">
<a href="/xmlui/handle/1866/3010" role="menuitem">Faculté des arts et des sciences</a>
</li>
<li role="presentation">
<a href="/xmlui/handle/1866/2942" role="menuitem">Faculté des arts et des sciences – Département de chimie</a>
</li>
<li role="presentation">
<a href="/xmlui/handle/1866/2977" role="menuitem">Faculté des arts et des sciences – Département de chimie – Thèses et mémoires</a>
</li>
<li class="disabled" role="presentation">
<a href="#" role="menuitem">View Item</a>
</li>
</ul>
</div>
<ul class="breadcrumb hidden-xs">
<li>
<i aria-hidden="true" class="glyphicon glyphicon-home"></i>  <a href="/xmlui/">Home</a>
</li>
<li>
<a href="/xmlui/handle/1866/3010">Faculté des arts et des sciences</a>
</li>
<li>
<a href="/xmlui/handle/1866/2942">Faculté des arts et des sciences – Département de chimie</a>
</li>
<li>
<a href="/xmlui/handle/1866/2977">Faculté des arts et des sciences – Département de chimie – Thèses et mémoires</a>
</li>
<li class="active">View Item</li>
</ul>
</div>
</div>
</div>
</div>
<div class="hidden" id="no-js-warning-wrapper">
<div id="no-js-warning">
<div class="notice failure">JavaScript is disabled for your browser. Some features of this site may not work without it.</div>
</div>
</div>
<div class="container" id="main-container">
<div class="row row-offcanvas row-offcanvas-left">
<div class="horizontal-slider clearfix">
<div class="col-xs-6 col-sm-3 sidebar-offcanvas" id="sidebar" role="navigation">
<div class="word-break hidden-print" id="ds-options">
<div class="ds-option-set" id="ds-search-option">
<form action="/xmlui/discover" class="" id="ds-search-form" method="post">
<fieldset>
<div class="input-group">
<input class="ds-text-field form-control" name="query" placeholder="Search" type="text"/><span class="input-group-btn"><button class="ds-button-field btn" title="Go"><span aria-hidden="true" class="glyphicon glyphicon-search"></span></button></span>
</div>
<div class="radio">
<label><input checked="" id="ds-search-form-scope-all" name="scope" type="radio" value=""/>Search Papyrus</label>
</div>
<div class="radio">
<label><input id="ds-search-form-scope-container" name="scope" type="radio" value="1866/2977"/>Search this Collection</label>
</div>
</fieldset>
</form>
</div>
<h2 class="ds-option-set-head h6">My Account</h2>
<div class="list-group" id="aspect_viewArtifacts_Navigation_list_account">
<div class="list-group-item ds-option details_Papyrus" id="aspect_eperson_Navigation_item_details_Papyrus">To submit an item or subscribe to email alerts.</div>
<div class="list-group-item ds-option">
<span class="bold"><a href="/xmlui/login">Login</a></span>
</div>
<a class="list-group-item ds-option" href="/xmlui/register">New user?</a>
</div>
<h2 class="ds-option-set-head h6">Browse</h2>
<div class="list-group" id="aspect_viewArtifacts_Navigation_list_browse">
<a class="list-group-item active"><span class="h5 list-group-item-heading h5">All of Papyrus</span></a><a class="list-group-item ds-option" href="/xmlui/community-list">Communities and Collections</a><a class="list-group-item ds-option" href="/xmlui/browse?type=title">Titles</a><a class="list-group-item ds-option" href="/xmlui/browse?type=dateissued">Issue Dates</a><a class="list-group-item ds-option" href="/xmlui/browse?type=author">Authors</a><a class="list-group-item ds-option" href="/xmlui/browse?type=advisor">Advisors</a><a class="list-group-item ds-option" href="/xmlui/browse?type=subject">Subjects</a><a class="list-group-item ds-option" href="/xmlui/browse?type=discipline">Disciplines</a><a class="list-group-item ds-option" href="/xmlui/browse?type=affiliation">Affiliation</a><a class="list-group-item ds-option" href="/xmlui/browse?type=titleindex">Titles index</a><a class="list-group-item active"><span class="h5 list-group-item-heading h5">This Collection</span></a><a class="list-group-item ds-option" href="/xmlui/handle/1866/2977/browse?type=title">Titles</a><a class="list-group-item ds-option" href="/xmlui/handle/1866/2977/browse?type=dateissued">Issue Dates</a><a class="list-group-item ds-option" href="/xmlui/handle/1866/2977/browse?type=author">Authors</a><a class="list-group-item ds-option" href="/xmlui/handle/1866/2977/browse?type=advisor">Advisors</a><a class="list-group-item ds-option" href="/xmlui/handle/1866/2977/browse?type=subject">Subjects</a><a class="list-group-item ds-option" href="/xmlui/handle/1866/2977/browse?type=discipline">Disciplines</a><a class="list-group-item ds-option" href="/xmlui/handle/1866/2977/browse?type=affiliation">Affiliation</a><a class="list-group-item ds-option" href="/xmlui/handle/1866/2977/browse?type=titleindex">Titles index</a>
</div>
<div class="list-group" id="aspect_viewArtifacts_Navigation_list_context"></div>
<div class="list-group" id="aspect_viewArtifacts_Navigation_list_administrative"></div>
<div class="list-group" id="aspect_discovery_Navigation_list_discovery"></div>
<h2 class="ds-option-set-head h6">Statistics</h2>
<div class="list-group" id="aspect_statistics_Navigation_list_statistics">
<a class="list-group-item ds-option" href="/xmlui/handle/1866/21949/statistics">View Usage Statistics</a>
</div>
</div>
</div>
<div class="col-xs-12 col-sm-12 col-md-9 main-content">
<div>
<div class="ds-static-div primary" id="aspect_artifactbrowser_ItemViewer_div_item-view">
<p class="ds-paragraph item-view-toggle item-view-toggle-top">
<span class="item-view-toggle"><a href="/xmlui/handle/1866/21949">Show item record</a></span>
</p>
<!-- External Metadata URL: cocoon://metadata/handle/1866/21949/mets.xml-->
<h2 class="page-header first-page-header">Accélération de l'exploration de l'espace chimique du cytochrome P450 BM3 par des méthodes de criblage à haut débit et bio-informatiques</h2>
<div class="ds-table-responsive">
<table class="ds-includeSet-table detailtable table table-striped table-hover">
<tr class="ds-table-row odd">
<td class="label-cell">dc.contributor.advisor</td><td class="word-break">Pelletier, Joelle</td><td></td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dc.contributor.author</td><td class="word-break">Rousseau, Olivier</td><td></td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dc.date.accessioned</td><td class="word-break">2019-05-24T16:50:00Z</td><td></td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dc.date.available</td><td class="word-break">NO_RESTRICTION</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dc.date.available</td><td class="word-break">2019-05-24T16:50:00Z</td><td></td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dc.date.issued</td><td class="word-break">2019-03-13</td><td></td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dc.date.submitted</td><td class="word-break">2018-09</td><td></td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dc.identifier.uri</td><td class="word-break">http://hdl.handle.net/1866/21949</td><td></td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dc.subject</td><td class="word-break">Analyse de composantes indépendantes du temps et de la structure</td><td>fr</td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dc.subject</td><td class="word-break">analyse statistique</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dc.subject</td><td class="word-break">automatisation</td><td>fr</td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dc.subject</td><td class="word-break">biocatalyse</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dc.subject</td><td class="word-break">cétone de framboise</td><td>fr</td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dc.subject</td><td class="word-break">criblage à haut débit</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dc.subject</td><td class="word-break">cytochrome P450 BM3</td><td>fr</td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dc.subject</td><td class="word-break">dynamique moléculaire</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dc.subject</td><td class="word-break">indigo</td><td>fr</td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dc.subject</td><td class="word-break">ingénierie de protéines</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dc.subject</td><td class="word-break">modélisation</td><td>fr</td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dc.subject</td><td class="word-break">mutagénèse</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dc.subject</td><td class="word-break">oxydation</td><td>fr</td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dc.subject</td><td class="word-break">séquençage nouvelle génération</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dc.subject</td><td class="word-break">Automation</td><td>fr</td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dc.subject</td><td class="word-break">Biocatalysis</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dc.subject</td><td class="word-break">High-throughput screening</td><td>fr</td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dc.subject</td><td class="word-break">Modeling</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dc.subject</td><td class="word-break">Molecular dynamics</td><td>fr</td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dc.subject</td><td class="word-break">Mutagenesis</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dc.subject</td><td class="word-break">Next-generation sequencing</td><td>fr</td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dc.subject</td><td class="word-break">Oxidation</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dc.subject</td><td class="word-break">Protein engineering</td><td>fr</td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dc.subject</td><td class="word-break">Raspberry ketone</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dc.subject</td><td class="word-break">Statistical analysis</td><td>fr</td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dc.subject</td><td class="word-break">Time-structure independent component analysis</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dc.subject.other</td><td class="word-break">Chemistry - Biochemistry / Chimie - Biochimie (UMI : 0487)</td><td>fr</td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dc.title</td><td class="word-break">Accélération de l'exploration de l'espace chimique du cytochrome P450 BM3 par des méthodes de criblage à haut débit et bio-informatiques</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dc.type</td><td class="word-break">Thèse ou mémoire / Thesis or Dissertation</td><td></td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">etd.degree.discipline</td><td class="word-break">Chimie</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">etd.degree.grantor</td><td class="word-break">Université de Montréal</td><td>fr</td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">etd.degree.level</td><td class="word-break">Maîtrise / Master's</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">etd.degree.name</td><td class="word-break">M. Sc.</td><td>fr</td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dcterms.abstract</td><td class="word-break">L’application de la chimie organique est prépondérante dans plusieurs secteurs industriels tels que le pharmaceutique, le cosmétique, l’alimentaire ou les produits ménagers. La chimie organique évolue sans cesse pour rendre les synthèses plus efficaces et moins dispendieuses. Cependant, l’amélioration de voies de synthèse se fait souvent sans égard à leur effet sur l’environnement. Ceci a entrainé l’émergence de la chimie verte pour réduire leur impact environnemental. À cette fin, l’une des stratégies employées est la biocatalyse : l’utilisation d’enzymes due à leurs propriétés ‘vertes’ et leur grande efficacité réactionnelle. 
Les oxydases de type cytochrome P450 sont reconnues pour leur grande promiscuité, liée à leur rôle dans la détoxification de composés xénobiotiques et de molécules endogènes dans le corps. Cette superfamille d’enzymes accomplit une diversité de réactions d’oxydations en une seule étape telles que l’hydroxylation, l’époxidation, la désamination, la déshalogénation et la cyclopropanation. La P450 BM3 (P450 BM3) de Bacillus megaterium possède l’une des plus grandes activités catalytiques de sa famille, justifiant l’intérêt grandissant de ses applications industrielles. Ce mémoire se penche sur la nécessité d’améliorer les essais afin d’augmenter la capacité de criblage pour obtenir une plus grande proportion de variants  capable de catalyser une réaction désirée. 
Il a été proposé que la synthèse de l’indigo par les variants de P450 BM3 soit un indicateur de promiscuité envers de nouveaux substrats. Nous avons exploré l’espace mutationnel de P450 BM3 pour la synthèse de l’indigo en comparant différentes méthodes de criblage de librairies de variants.  Dans un premier temps, des librairies de variants furent créées par mutagénèse par saturation de site. Cela permet d’obtenir une représentation uniforme des variants à chaque position sélectionnée plus rapidement et à un coût modique. Ensuite, des méthodes de criblage furent optimisées pour quantifier les variants de P450 BM3 et les cribler de deux façons. Nous avons comparé un essai colorimétrique direct grâce à l’absorbance de l’indigo avec un essai général indirect combinant la fluorescence du cofacteur NADPH et la spectrométrie de masse (LC-MS, GC-MS et LC préparative). Cette deuxième approche permet de cribler les variants indépendamment du substrat. Afin d’augmenter le débit du criblage, nous avons optimisé les méthodes en utilisant des plaques 96 puits avec une station de pipetage automatisée. Nous avons découvert de multiples nouveaux variants de P450 BM3 synthétisant l’indigo dont une forte fraction de ces derniers s’est révélée aptes à synthétiser une molécule de valeur élevée en industrie : la cétone de framboise. 
La corrélation de l’activité de P450 BM3 avec l’indigo et la cétone de framboise justifie l’exploration plus profonde de l’espace mutationnel de P450 BM3. La compréhension de cette corrélation permettrait de guider d’avantage l’ingénierie de P450 BM3 vers de nouveaux substrats. À cette fin, nous tentons d’expliquer la relation entre la structure, le dynamisme et l’activité des variants de P450 BM3. Les méthodes de criblage conventionnelles ne sont pas assez efficaces pour générer de grandes séries de données (« Big Data »). Nous appliquons le séquençage de nouvelle génération afin d’identifier &gt;104 variants simultanément pour relier le génotype (ADN) au phénotype (synthèse d’indigo). L’analyse de ces données nécessite l’utilisation d’outils statistiques et informatiques tels que la modélisation, la dynamique moléculaire et l’analyse de composantes indépendantes du temps et de la structure (tICA). Nous observons une tendance commune des variants actifs de P450 BM3 représenté par un changement d’équilibre entre ses deux conformations majeures.  Ce changement conformationnel pourrait expliquer la propension de P450 BM3 à synthétiser l’indigo.
Ce mémoire présente ainsi des méthodes permettant d’accélérer la découverte de variants actifs de P450 BM3 pour diverses réactions d’intérêt en industrie. Les méthodologies ci-développées pourront ensuite guider l’ingénierie d’autres systèmes enzymatiques.</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">dcterms.abstract</td><td class="word-break">Organic chemistry plays a major role in the pharmaceutical, cosmetic, food and household product industries. Industrial organic synthesis has constantly evolved to be more efficient and less cost-effective. However, those improvements were often made regardless of their impact on the environment, spawning the creation of the discipline of Green Chemistry. One among many strategies to make chemistry more innocuous to the environment is to apply biocatalysis: the use of enzymes for their ‘green’ properties and catalytic efficacy.
Cytochrome P450 oxidases are well known for their substrate promiscuity in detoxification of xenobiotics and endogenous molecules in the body. This superfamily of enzymes catalyzes a diversity of oxidation reactions in a single step, including hydroxylations, epoxidations, deaminations, dehalogenations and cyclopropanations. Cytochrome P450 BM3 (P450 BM3) from Bacillus megaterium is one of the most efficient of its family, justifying the growing interest for its industrial applications. This thesis considers the necessity to improve assays in order to increase the throughput of enzyme library screening and to improve the ratio of active variants.
Indigo synthesis has been proposed to be an efficient predictor of promiscuity towards novel substrates of P450 BM3. We explore the mutational space of P450 BM3 for indigo synthesis and compare screening methods. We made variant libraries using site-directed saturation mutagenesis to obtain a uniform distribution of mutations at each position and reduce cost and time. We optimized a method to quantify P450 BM3 variants and two different screening methods. We compared a direct colorimetric assay using indigo absorbance and an indirect assay using NADPH fluorescence coupled with mass spectrometry (LC-MS, GC-MS and LC prep). This second approach circumvents the need to develop an assay dependent of the substrate. To increase screening throughput, we optimized methods in a 96-well plate format and use an automated pipetting station. We confirm the discovery of new indigo-producing variants. We further demonstrate that indigo can serve as a strong predictor of raspberry ketone production, a high value industrial compound.

The correlation of the activity of P450 BM3 with indigo and raspberry ketone justifies the exploration of the fitness landscape P450 BM3. Understanding this correlation would accelerate engineering of P450 BM3 for reaction with novel substrates. We attempt to explain the relationship between the structure, dynamism and activity through deep-mutational scanning of P450 BM3. Conventional screening methods are not sufficient to produce large datasets (Big Data); we apply next-generation sequencing (NGS) to identify &gt;104 variants simultaneously and to link the genotype (DNA) and the phenotype (indigo production).  Analysis of NGS for enzyme engineering required the development of statistical and bioinformatics tools such as modeling, molecular dynamics and time-structure independent component analysis (tICA). Those methods allowed us to observe conformational perturbations that were shared between the active variants represented as a shift of equilibrium between two major conformations.  This shift may explain the propensity of diverse P450 BM3 variants to synthesize indigo.
This master’s thesis successfully demonstrates the accelerated discovery of engineered variants of P450 BM3 with activity for industrially relevant reactions. The methodologies we have developed contribute to knowledge on enzyme engineering and have the potential to be applied more broadly in other enzyme systems.</td><td>fr</td>
</tr>
<tr class="ds-table-row even">
<td class="label-cell">dcterms.language</td><td class="word-break">fra</td><td>fr</td>
</tr>
<tr class="ds-table-row odd">
<td class="label-cell">UdeM.ORCIDAuteurThese</td><td class="word-break">0000-0002-8691-2048</td><td>fr</td>
</tr>
</table>
</div>
<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Adc&amp;rft_id=http%3A%2F%2Fhdl.handle.net%2F1866%2F21949&amp;rfr_id=info%3Asid%2Fdspace.org%3Arepository&amp;rft.degree=Chimie&amp;rft.degree=Universit%C3%A9+de+Montr%C3%A9al&amp;rft.degree=Ma%C3%AEtrise+%2F+Master%27s&amp;rft.degree=M.+Sc."> ﻿ 
        </span>
<h3>Files in this item</h3>
<div class="file-list">
<div class="file-wrapper row">
<div class="col-xs-6 col-sm-3">
<div class="thumbnail-wrapper">
<a class="image-link" href="/xmlui/bitstream/handle/1866/21949/Rousseau_Olivier_2018_memoire.pdf?sequence=2&amp;isAllowed=y"><img alt="Thumbnail" class="thumbnail-papyrus" src="/xmlui/bitstream/handle/1866/21949/Rousseau_Olivier_2018_memoire.pdf.jpg?sequence=4&amp;isAllowed=y"/></a>
<div class="imageoverlay">
<img height="49" src="/xmlui/themes/Mirage2/images/mimes/mime_icone_pdf.png" style="height: 49px;" width="44"/></div>
</div>
</div>
<div class="col-xs-6 col-sm-7">
<dl class="file-metadata dl-horizontal">
<dt>Name:</dt>
<dd class="word-break" title="Rousseau_Olivier_2018_memoire.pdf">
<a href="/xmlui/bitstream/handle/1866/21949/Rousseau_Olivier_2018_memoire.pdf?sequence=2&amp;isAllowed=y">Rousseau_Olivier_2018_memoire.pdf</a>
</dd>
<dt>Size:</dt>
<dd class="word-break">4.913Mb</dd>
<dt>Format:</dt>
<dd class="word-break">PDF</dd>
<dt>Description:</dt>
<dd class="word-break" title="Mémoire">Mémoire</dd>
</dl>
</div>
</div>
</div>
<h3 class="ds-list-head">This item appears in the following Collection(s)</h3>
<ul class="ds-referenceSet-list">
<!-- External Metadata URL: cocoon://metadata/handle/1866/2621/mets.xml-->
<li>
<a href="/xmlui/handle/1866/2621">Thèses et mémoires électroniques de l’Université de Montréal</a> [18270]<br/>
</li>
<!-- External Metadata URL: cocoon://metadata/handle/1866/2977/mets.xml-->
<li>
<a href="/xmlui/handle/1866/2977">Faculté des arts et des sciences – Département de chimie – Thèses et mémoires</a> [594]<br/>
</li>
</ul>
<p class="ds-paragraph item-view-toggle item-view-toggle-bottom">
<span class="item-view-toggle"><a href="/xmlui/handle/1866/21949">Show item record</a></span>
</p>
</div>
</div>
<div class="visible-xs visible-sm">
<footer>
<div class="row">
<hr/>
<div class="col-xs-7 col-sm-8">
<div>
<a href="http://www.dspace.org/" target="_blank">DSpace software</a>
                        [version 5.8 XMLUI],
                        copyright © 2002-2015  <a href="http://www.duraspace.org/" target="_blank">DuraSpace</a>
</div>
<div class="hidden-print">
<a href="/xmlui/contact">Contact Us</a> | <a href="/xmlui/feedback">Send Feedback</a>
</div>
</div>
<div class="col-xs-5 col-sm-4 hidden-print">
<div class="pull-right">
<script type="text/javascript"> 
										      (function(d, t) { 
										        var s = d.createElement(t), options = {'domain':'papyrus.bib.umontreal.ca','style': '16','container':'entrust-net-seal'}; 
										        s.src = 'https://seal.entrust.net/sealv2.js'; 
										        s.async = true; 
										        var scr = d.getElementsByTagName(t)[0], par = scr.parentNode; par.insertBefore(s, scr); 
										        s.onload = s.onreadystatechange = function() { 
										        var rs = this.readyState; if (rs) if (rs != 'complete') if (rs != 'loaded') return; 
										        try{goEntrust(options)} catch (e) {} }; 
										        })(document, 'script'); 
										</script>
<div id="entrust-net-seal">
<a href="https://www.entrust.com/ssl-certificates/">Certificat SSL / SSL Certificate</a>
</div>
</div>
</div>
</div>
<div class="row bib-footer hidden-print">
<div class="col-xs-12 col-sm-12 col-md-4">
<a class="footerEXLlink" href="http://www.bib.umontreal.ca"><img alt="les bibliothèques/UdeM" src="/xmlui/themes/Mirage2/images/propulse-par.png"/></a>
</div>
<div class="hidden-xs col-sm-12 col-md-8">
<nav>
<ul>
<li>
<a href="https://www.urgence.umontreal.ca/">Emergency</a>
</li>
<li>
<a href="http://www.carrieres.umontreal.ca/">Careers</a>
</li>
<li>
<a href="https://outlook.umontreal.ca/">My email</a>
</li>
<li>
<a href="https://studium.umontreal.ca/">StudiUM</a>
</li>
<li>
<a href="http://itunesu.umontreal.ca/">iTunes U</a>
</li>
<li>
<a href="http://www.bib.umontreal.ca/a-propos/nous-ecrire.htm">Contact us</a>
</li>
<li>
<a href="https://www.facebook.com/bibUdeM"><img alt="Facebook" src="/xmlui/themes/Mirage2/images/bib-umontreal/logo-Facebook.png"/></a>
</li>
<li>
<a href="https://www.youtube.com/user/BibliothequesUdeM"><img alt="YouTube" src="/xmlui/themes/Mirage2/images/bib-umontreal/logo-YouTube.png"/></a>
</li>
<li>
<a href="https://twitter.com/bibUdeM"><img alt="Twitter" src="/xmlui/themes/Mirage2/images/bib-umontreal/logo-Twitter.png"/></a>
</li>
<li>
<a href="https://www.nouvelles.umontreal.ca/frontpage/rss.html"><img alt="University RSS" src="/xmlui/themes/Mirage2/images/bib-umontreal/icone-syndication-14x14.png"/></a>
</li>
</ul>
</nav>
</div>
</div>
<a class="hidden" href="/xmlui/htmlmap"> </a>
<p> </p>
</footer>
</div>
</div>
</div>
</div>
<div class="hidden-xs hidden-sm">
<footer>
<div class="row">
<hr/>
<div class="col-xs-7 col-sm-8">
<div>
<a href="http://www.dspace.org/" target="_blank">DSpace software</a>
                        [version 5.8 XMLUI],
                        copyright © 2002-2015  <a href="http://www.duraspace.org/" target="_blank">DuraSpace</a>
</div>
<div class="hidden-print">
<a href="/xmlui/contact">Contact Us</a> | <a href="/xmlui/feedback">Send Feedback</a>
</div>
</div>
<div class="col-xs-5 col-sm-4 hidden-print">
<div class="pull-right">
<script type="text/javascript"> 
										      (function(d, t) { 
										        var s = d.createElement(t), options = {'domain':'papyrus.bib.umontreal.ca','style': '16','container':'entrust-net-seal'}; 
										        s.src = 'https://seal.entrust.net/sealv2.js'; 
										        s.async = true; 
										        var scr = d.getElementsByTagName(t)[0], par = scr.parentNode; par.insertBefore(s, scr); 
										        s.onload = s.onreadystatechange = function() { 
										        var rs = this.readyState; if (rs) if (rs != 'complete') if (rs != 'loaded') return; 
										        try{goEntrust(options)} catch (e) {} }; 
										        })(document, 'script'); 
										</script>
<div id="entrust-net-seal">
<a href="https://www.entrust.com/ssl-certificates/">Certificat SSL / SSL Certificate</a>
</div>
</div>
</div>
</div>
<div class="row bib-footer hidden-print">
<div class="col-xs-12 col-sm-12 col-md-4">
<a class="footerEXLlink" href="http://www.bib.umontreal.ca"><img alt="les bibliothèques/UdeM" src="/xmlui/themes/Mirage2/images/propulse-par.png"/></a>
</div>
<div class="hidden-xs col-sm-12 col-md-8">
<nav>
<ul>
<li>
<a href="https://www.urgence.umontreal.ca/">Emergency</a>
</li>
<li>
<a href="http://www.carrieres.umontreal.ca/">Careers</a>
</li>
<li>
<a href="https://outlook.umontreal.ca/">My email</a>
</li>
<li>
<a href="https://studium.umontreal.ca/">StudiUM</a>
</li>
<li>
<a href="http://itunesu.umontreal.ca/">iTunes U</a>
</li>
<li>
<a href="http://www.bib.umontreal.ca/a-propos/nous-ecrire.htm">Contact us</a>
</li>
<li>
<a href="https://www.facebook.com/bibUdeM"><img alt="Facebook" src="/xmlui/themes/Mirage2/images/bib-umontreal/logo-Facebook.png"/></a>
</li>
<li>
<a href="https://www.youtube.com/user/BibliothequesUdeM"><img alt="YouTube" src="/xmlui/themes/Mirage2/images/bib-umontreal/logo-YouTube.png"/></a>
</li>
<li>
<a href="https://twitter.com/bibUdeM"><img alt="Twitter" src="/xmlui/themes/Mirage2/images/bib-umontreal/logo-Twitter.png"/></a>
</li>
<li>
<a href="https://www.nouvelles.umontreal.ca/frontpage/rss.html"><img alt="University RSS" src="/xmlui/themes/Mirage2/images/bib-umontreal/icone-syndication-14x14.png"/></a>
</li>
</ul>
</nav>
</div>
</div>
<a class="hidden" href="/xmlui/htmlmap"> </a>
<p> </p>
</footer>
</div>
</div>
<script src="https://www.google.com/jsapi"> </script><script>if(!window.DSpace){window.DSpace={};}window.DSpace.context_path='/xmlui';window.DSpace.theme_path='/xmlui/themes/Mirage2/';</script><script src="/xmlui/themes/Mirage2/scripts/theme.js"> </script><script>
                  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
                  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
                  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

                  ga('create', 'UA-305727-1', 'papyrus.bib.umontreal.ca');
                  ga('send', 'pageview');
           </script><script>         
             var details_Papyrus =  $("#aspect_viewArtifacts_Navigation_list_account .details_Papyrus").text();
             $("#aspect_viewArtifacts_Navigation_list_account a[href='/xmlui/login']").attr("title",details_Papyrus);
        </script>
</body></html>
